index.html

```diff
@@ -13,7 +13,9 @@
 London<br>
 Paris<br>
 Tokyo<br>
-Pune
+Pune<br>
+Mumbai<br>
+Delhi
```

Comment:
@bhaveshf-cuelogic : I have made the changes, please review it once more. thanks